src/Cake.Common/Tools/OctopusDeploy/PushPackageSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
```

Comment:
@agc93 from a consistency stand point, this extra line should be removed here.  Personally, I think we do need a linebreak here, but let's keep things the same for now.